terraform-build-user/main.tf

```
@@ -9,6 +9,6 @@ module "iam_user" {
     aws.images-staging-ssm    = aws.images-staging-ssm
   }
 
-  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*"]
+  ssm_parameters = ["/windows/server/administrator/password"]
   user_name      = "build-windows-commando-packer"
```

The username needs to change here as well:
```suggestion
  user_name      = "build-windows-server-packer"
```

Once that is done, I can create the build user for you.

---

@dav3r resolved with [64ddc25](https://github.com/cisagov/windows-server-packer/pull/10/commits/64ddc25652ddd869f7b6353ab768792c1e7176e4)